openvpn-install.sh

```diff
@@ -44,8 +44,24 @@ elif [[ -e /etc/centos-release || -e /etc/redhat-release ]]; then
 	RCLOCAL='/etc/rc.d/rc.local'
 	# Needed for CentOS 7
 	chmod +x /etc/rc.d/rc.local
+elif [[ -e /etc/arch-release ]]; then
+	OS=arch
+	RCLOCAL='/etc/rc.local'
+	# Needed for rc.local support on ArchLinux
+	echo "[Unit]
```

Comment:
You confirm that this add a functionnal rc.local, and therefore resolving https://github.com/Angristan/OpenVPN-install/issues/1 ?